### ISSUE TITLE
[help needed] Couchbase lite version ?

### ISSUE THREAD

**Issue by rajaishwary** [couchbaselabs/react-native-couchbase-lite#129]
Which couchbase lite version this repo supports ?

**Comment by ajres** [comment]
@rajaishwary 

github issues are meant for reporting bugs, if you still need help with this question please ask post over on the [couchbase forums](https://forums.couchbase.com).

**Comment by rajaishwary** [comment]
@ajres Yes, I understand that but at that point of time i needed to now which docs should i prefer for rest apis.I digg in to the repo and got my answer.
You can close the issue.
